src/app/main/component/eco-news/components/remaining-count/remaining-count.component.ts

```diff
@@ -1,10 +1,23 @@
 import { Component, Input } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
 
 @Component({
   selector: 'app-remaining-count',
   templateUrl: './remaining-count.component.html',
-  styleUrls: ['./remaining-count.component.scss'],
+  styleUrls: ['./remaining-count.component.scss']
 })
 export class RemainingCountComponent {
   @Input() public remainingCount = 0;
+
+  constructor(public translate: TranslateService) {}
+
+  setWordDeclension(count: string): string {
+    if (count.slice(-1) === '1' && count.slice(-2) !== '11') {
+      return 'ends_on_1';
+    } else if ('234'.includes(count.slice(-1)) && !['12', '13', '14'].includes(count.slice(-2))) {
+      return 'ends_on_2_3_4';
+    } else {
+      return '';
+    }
+  }
```

Comment:
Remove 'if else' statement (you use return, change to 'if' 'if')
IF(){RETURN}
IF(){RETURN}
RETURN
Also, all numbers except 1 0 -1 are considered - magic numbers, so you should make them consts.

const lastTwo = -2;
And good practice on using switch case is to make the keys more compact like 1,2,3 or a,b,c not 'ends_on_2_3_4'

Reply:
Done. Thanks a lot!